hack/e2e-kueueviz-local.sh

```diff
@@ -21,6 +21,8 @@ ROOT_DIR="$SOURCE_DIR/.."
 # shellcheck source=hack/e2e-common.sh
 source "${SOURCE_DIR}/e2e-common.sh"
 
+KUEUE_NS="${KUEUE_NAMESPACE:-kueue-system}"
```

Comment:
Why do we need this one?
Don't we have the same environment variable in `e2e-common.sh`?